src/bicep/modules/defender.bicep

```diff
@@ -16,15 +18,18 @@ var bundle = (environment().name != 'AzureUSGovernment' ? [
   'AppServices'
   'Dns'
   'Arm'
-  ] : [
-  'SqlServers'
-  'VirtualMachines'
-  'StorageAccounts'
-  'ContainerRegistry'
-  'KubernetesService'
-  'Dns'
-  'Arm'
-])
+  ] : (envName == 'AzureUSGovernment') ? [
+    'SqlServers'
+    'VirtualMachines'
+    'StorageAccounts'
+    'ContainerRegistry'
+    'KubernetesService'
+    'Dns'
+    'Arm'
+  ] : []
+
+output bundle array = bundle
```

Comment:
I don't think we use this output anywhere and it can be removed

Reply:
Good point! 
Updated in commit [e3791a3](https://github.com/Azure/missionlz/pull/687/commits/e3791a317900377d4967feddccb7ecb6b357a1c3)